### ISSUE TITLE
Views that have modifiable Values should get their name from a nearby Label.

### ISSUE THREAD

**Issue by mgifford** [cds-snc/covid-alert-app#1070]
# Summary

Axe on mobile reports that when submitting the one-time key that the field is missing a label. 

# Steps to reproduce

1. Run axe on your mobile device
2. Go to Enter your one-time key page
3. Test page

# Environment | Environnement logiciel

Android.

# Screencapture
![50249704952_2804e42bc7_c](https://user-images.githubusercontent.com/116832/90831597-176a9980-e312-11ea-9e9e-3d4589723018.jpg)
